src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css

```diff
@@ -4,16 +4,15 @@
 }
 
 .label > *:not(:last-child, .itemsCount) {
-  margin-right: 10px;
+  margin-right: 12px;
 }
 
 .labelText {
-  font-size: 12px;
   line-height: 1;
 }
 
 .labelTextStrong {
-  font-size: 12px;
+  font-size: 13px;
```

Comment:
Are you sure this font-size rule is necessary? I unticked it in dev tools, and didn't see any change in the track panel.

Reply:
No we don't need this. I forgot to remove this one. Thanks for spotting it.